core/getNodeFromDBErrWithKey.go

```diff
@@ -0,0 +1,50 @@
+package core
+
+import (
+	"encoding/hex"
+	"fmt"
+)
+
+// GetNodeFromDBErrorString represents the string which is returned when a getting node from DB returns an error
+const GetNodeFromDBErrorString = "getNodeFromDB error"
+
+// GetNodeFromDBErrWithKey defines a custom error for trie get node
```

Comment:
start comment with lower case to align with the function name?

Reply:
Changed.